src/test/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsControllerTests.java

```diff
@@ -57,18 +57,25 @@ public class UserCommonsControllerTests extends ControllerTestCase {
   @Autowired
   private ObjectMapper objectMapper;
 
+  public static Commons dummyCommons(long id) {
+    Commons commons = new Commons(id, "test", 1, 1, 1, LocalDateTime.now(), 1, true, 1, 1, new ArrayList<User>());
+    return commons;
+  }
+
   public static UserCommons dummyUserCommons(long id) {
     UserCommons userCommons = new UserCommons(id,1,1,"test",1,1, 100);
     return userCommons;
   }
   @WithMockUser(roles = { "ADMIN" })
   @Test
   public void test_getUserCommonsById_exists_admin() throws Exception {
-  
+
+    Commons dummyCommons = dummyCommons(1);
+    when(commonsRepository.findById(1L)).thenReturn(Optional.of(dummyCommons));
     UserCommons expectedUserCommons = dummyUserCommons(1);
     when(userCommonsRepository.findByCommonsIdAndUserId(eq(1L),eq(1L))).thenReturn(Optional.of(expectedUserCommons));
 
-    MvcResult response = mockMvc.perform(get("/api/usercommons/?userId=1&commonsId=1"))
+    MvcResult response = mockMvc.perform(get("/api/usercommons?commonsId=1&userId=1"))
```

Comment:
any reason to swap these?

Reply:
also may still need the forward slash after usercommons

Reply:
I needed this to pass tests, the call is slightly different